stock_location_orderpoint/tests/test_sublocation_orderpoint.py

```diff
@@ -0,0 +1,99 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from .common import TestLocationOrderpointCommon
+
+
+class TestSubLocationOrderpoint(TestLocationOrderpointCommon):
```

Comment:
@rousseldenis This file is not imported into __init__ and tests are therefore not executed by CI Moreover, the tests are failing.